tests/travis/_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I copied this exact travis folder from `1.0`. Is there a reason you didn't use it? :P